### ISSUE TITLE
Issues with streaming on gstreamer.

### ISSUE THREAD

**Issue by ajay-collab** [aler9/rtsp-simple-server#737]
Current OS: Ubuntu 21.10
Took the latest release file and started the rtsp server without any changes, the stream gets accessible on ffmpeg but not gstreamer has trouble accessing the stream. It crashes with the following log, 
`gst-play-1.0 rtsp://localhost:8554/mystream
Press 'k' to see a list of keyboard shortcuts.
Now playing rtsp://localhost:8554/mystream
Pipeline is live.
ERROR Could not get/set settings from/on resource. for rtsp://localhost:8554/mystream
ERROR debug information: ../gst/udp/gstmultiudpsink.c(1225): gst_multiudpsink_configure_client (): /GstPlayBin:playbin/GstURIDecodeBin:uridecodebin0/GstRTSPSrc:source/GstUDPSink:udpsink0:
Invalid address family (got 10)
Reached end of play list.

(gst-play-1.0:11695): GStreamer-CRITICAL **: 13:26:40.793: 
Trying to dispose element playbin, but it is in PAUSED instead of the NULL state.
You need to explicitly set elements to the NULL state before
dropping the final reference, to allow them to clean up.
This problem may also be caused by a refcounting bug in the
application or some element.
`

Any idea why there's an issue on ubuntu 21, 
the GStreamer Core Library version 1.18.

**Comment by aler9** [comment]
Hello, please provide server logs.

**Comment by ajay-collab** [comment]
```
2021/12/23 12:16:39 [I] [0/0/0] rtsp-simple-server v0.15.5
2021/12/23 12:16:39 [I] [0/0/0] [RTSP/UDP/RTP listener] opened on :8000
2021/12/23 12:16:39 [I] [0/0/0] [RTSP/UDP/RTCP listener] opened on :8001
2021/12/23 12:16:39 [I] [0/0/0] [RTSP/TCP listener] opened on :8554
2021/12/23 12:16:39 [I] [0/0/0] [RTMP listener] opened on :1935
2021/12/23 12:17:05 [I] [1/0/0] [client 127.0.0.1:52202] connected (RTSP/TCP)
2021/12/23 12:17:05 [I] [1/1/0] [client 127.0.0.1:52202] is publishing to path 'mystream', 1 track with udp
2021/12/23 12:17:18 [I] [2/1/0] [client [::1]:39250] connected (RTSP/TCP)
2021/12/23 12:17:19 [I] [2/1/1] [client [::1]:39250] is reading from path 'mystream', 1 track with udp

```

**Comment by aler9** [comment]
Please try again with the latest version (v0.17.12), we don't support issues with old versions (v0.15.5).

**Comment by aler9** [comment]
This issue can be solved by using 127.0.0.1 instead of localhost. Anyway, it's a Gstreamer issue, not a server one.
